roles/openshift_node/meta/main.yml

```diff
@@ -13,4 +13,5 @@ galaxy_info:
   - cloud
 dependencies:
 - { role: openshift_common }
```

Comment:
openshift_common already has a stated dependency on openshift_repos, we shouldn't need it here as well.